MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/offline/SimpleOfflineMapActivity.java

```diff
@@ -158,30 +158,32 @@ protected void onStop() {
   public void onPause() {
     super.onPause();
     mapView.onPause();
-    offlineManager.listOfflineRegions(new OfflineManager.ListOfflineRegionsCallback() {
-      @Override
-      public void onList(OfflineRegion[] offlineRegions) {
-        if (offlineRegions.length > 0) {
-          // delete the last item in the offlineRegions list which will be yosemite offline map
-          offlineRegions[(offlineRegions.length - 1)].delete(new OfflineRegion.OfflineRegionDeleteCallback() {
-            @Override
-            public void onDelete() {
-              Toast.makeText(SimpleOfflineMapActivity.this, "Yosemite offline map deleted", Toast.LENGTH_LONG).show();
-            }
-
-            @Override
-            public void onError(String error) {
-              Log.e(TAG, "On Delete error: " + error);
-            }
-          });
+    if (offlineManager != null) {
+      offlineManager.listOfflineRegions(new OfflineManager.ListOfflineRegionsCallback() {
+        @Override
+        public void onList(OfflineRegion[] offlineRegions) {
+          if (offlineRegions.length > 0) {
+            // delete the last item in the offlineRegions list which will be yosemite offline map
+            offlineRegions[(offlineRegions.length - 1)].delete(new OfflineRegion.OfflineRegionDeleteCallback() {
+              @Override
+              public void onDelete() {
+                Toast.makeText(SimpleOfflineMapActivity.this, "Yosemite offline map deleted", Toast.LENGTH_LONG).show();
```

Comment:
Fixed the hardcoded string, note that we still have a bunch of hardcoded strings found in this example and others throughout the application.